Rakefile

```diff
@@ -36,7 +36,7 @@ def connect_db
   database = config['database']
 
   puts '[*] Connecting to DB'
-  conn = Mysql2::Client.new(:host => host, :username => user, :password => password, :database => database)
+  Mysql2::Client.new(:host host, username: user, password: password, database: database)
```

Comment:
@True-Demon you just forgot `host: host`

Have a nice day

Reply:
Lol, shortest commit of my life. XD
Sorry about that.